srcpkgs/fwupd/template

```diff
@@ -12,8 +12,9 @@ hostmakedepends="dejavu-fonts-ttf gnutls-tools pkg-config gcab gi-docgen vala
  protobuf-c-devel"
 makedepends="libxmlb-devel cairo-devel colord-devel libarchive-devel
  gnutls-devel gpgme-devel json-glib-devel libgusb-devel polkit-devel
```

Comment:
libgusb-devel can be dropped i think

Reply:
Packaging worked indeed fine without the `libgusb-devel` makedepends.